Configure

```diff
@@ -1237,7 +1237,10 @@ unless ($disabled{asm}) {
 
 my %predefined = compiler_predefined($target{cc});
 
-if (!$disabled{makedepend}) {
+# Check for makedepend capabilities.  Note that for VC- and vms- targets,
+# this should not be done, as it's likely to turn off a functionality that
+# actually exists, currently hard coded for cl (Windows) and CC/DECC (VMS).
+if ($config{target} !~ /^(VC|vms)-/ && !$disabled{makedepend}) {
```

Comment:
Wouldn't it be more natural to
```
if (!$disabled{makedepend}) {
    if ($predefined{__GNUC__} >= 3) {
    } elsif ($config{target} !~ /^VC|vms)/) {
    } else {
    }
}
```
I mean the message is "oh! user wants dependencies, see if compiler can do it for us", right? And it should read as such, not as "if non-windows-or-vms user asks us for dependencies...". I think former is more logical. It does mean that it would probably be appropriate to move "We know that GNU C version 3" commentary inside the first `if`, and add commentary about cl and vms compiler knowing what to do inside elsif...

Reply:
And one can as well make check for VC|vms first if...

Reply:
Yeah, ok, I can appreciate that logic.  Doing now.